user-manual/administer/dashboard-admin.rst

```diff
@@ -559,46 +558,83 @@ Clicking the date, name or UUID will display a report of the failure:
-uploaded using the web interface; they must exist on volumes accessible to the
-Archivematica MCP server and configured via the Storage Service.
+This section of the Administration page displays storage locations that have
+been configured for the Archivematica instance. Storage locations are configured
+in the :ref:`Storage Service <storageService:administrators>`. This table
+presents information from the Storage Service.
 
-When starting a transfer you are required to select one or more directories of
-files to add to the transfer.
+.. image:: images/dashboard-storage-locations.*
+   :align: center
+   :width: 80%
+   :alt: Table showing available storage locations configured for the Archivematica pipeline
 
-.. _dashboard-AIP-stor:
+The table on the Storage locations page displays the :ref:`purpose
+<location-purposes>` of the storage location, the location's description, how
+much space has been used, and the path.
 
-AIP storage locations
----------------------
+If a quota has not been set for the storage location, the location's available
+space will be set to *unlimited*. Quotas can be set for some types of locations
+through the :ref:`Storage Service <storageService:locations>`.
 
-AIP storage directories are directories in which completed AIPs are stored.
-Storage directories can be specified in a manner similar to transfer source
-directories using the Storage Service.
+*Used / available* values are not currently calculated for all types of storage
+locations, such as Replicator, Transfer Backlog, and Transfer Source locations.
 
-You can view your transfer source directories in the Administrative tab of the
-dashboard under "AIP storage locations".
+Depending on the configuration of your Archivematica deployment, you may not see
+values for all storage locations.
 
 .. _dashboard-usage:
 
 Processing storage usage
 ------------------------
 
-This section of the Administration page displays various processing locations
-with their current usage of available space.
+This section of the Administration page displays the Archivematica instance's
+processing locations. Processing locations are used to hold materials while they
+arebeing processed, as well as temporary storage directories for certain types
+of content - for example, failed and rejected transfers.
+
+.. important::
+
+   This page calculates usage statistics for Archivematica instances that have
+   been deployed with a shared directory on the same disk as the Archivematica
+   instance. If your Archivematica instance has been configured a different way,
+   the usage statistics may not be displayed.
+
+When you navigate to the page, click **Calculate disk usage** to see how much
+disk space is being used. The calculation will take a few minutes, and may take
+a long time depending on how your Archivematica instance is configured. Keep
+the tab open. You can continue to process materials while the disk usage is
+being calculated.
```

Comment:
This depends on a lot of things so I wouldn't say minutes. Also, you should be able to process materials while the calculation is running, it may not show the exact values after a while but that's hard to control. I'd only say "The calculation may take a long time depending on how your Archivematica instance is configured and the current disk usage".